src/VisualStudio/Core/Def/Commands.vsct

```diff
@@ -286,58 +282,6 @@
           <CommandName>ErrorListSetSeverityNone</CommandName>
         </Strings>
       </Button>
-      <!-- Buttons to set analysis scope for solution -->
-      <Button guid="guidRoslynGrpId" id="cmdidAnalysisScopeDefault" priority="100" type="Button">
-        <Parent guid="guidRoslynGrpId" id="grpAnalysisScopeItems" />
-        <CommandFlag>DefaultDisabled</CommandFlag>
-        <CommandFlag>TextChanges</CommandFlag>
-        <Strings>
-          <ButtonText>&amp;Default</ButtonText>
-          <CanonicalName>Default</CanonicalName>
-          <LocCanonicalName>Default</LocCanonicalName>
-          <CommandName>AnalysisScopeDefault</CommandName>
-        </Strings>
-      </Button>
-      <Button guid="guidRoslynGrpId" id="cmdidAnalysisScopeCurrentFile" priority="200" type="Button">
-        <Parent guid="guidRoslynGrpId" id="grpAnalysisScopeItems" />
-        <CommandFlag>DefaultDisabled</CommandFlag>
-        <Strings>
-          <ButtonText>&amp;Current Document</ButtonText>
```

Comment:
><ButtonText>&amp;Current Document</ButtonText> [](http://example.com/codeflow?start=10&length=46)

Are there resx strings to remove matching these too?

Reply:
will check!